pkg/provider/handlers/functions_test.go

```diff
@@ -107,3 +107,27 @@ func Test_findNamespace(t *testing.T) {
 		})
 	}
 }
+
+func Test_readMemoryLimitFromSpec(t *testing.T) {
+	type test struct {
+		Name     string
+		Spec     *specs.Spec
+		Expected int64
+	}
+	testLimit := int64(64)
+	tests := []test{
+		{Name: "specs.Linux not found", Spec: &specs.Spec{Linux: nil}, Expected: int64(0)},
+		{Name: "specs.LinuxResource not found", Spec: &specs.Spec{Linux: &specs.Linux{Resources: nil}}, Expected: int64(0)},
+		{Name: "specs.LinuxMemory not found", Spec: &specs.Spec{Linux: &specs.Linux{Resources: &specs.LinuxResources{Memory: nil}}}, Expected: int64(0)},
+		{Name: "specs.LinuxMemory.Limit not found", Spec: &specs.Spec{Linux: &specs.Linux{Resources: &specs.LinuxResources{Memory: &specs.LinuxMemory{Limit: nil}}}}, Expected: int64(0)},
+		{Name: "Memory limit set as expected", Spec: &specs.Spec{Linux: &specs.Linux{Resources: &specs.LinuxResources{Memory: &specs.LinuxMemory{Limit: &testLimit}}}}, Expected: int64(64)},
+	}
+	for _, tc := range tests {
+		t.Run(tc.Name, func(t *testing.T) {
+			got := readMemoryLimitFromSpec(tc.Spec)
+			if got != tc.Expected {
+				t.Fatalf("expected %d, got %d", tc.Expected, got)
```

Comment:
We tend to use the terminology "want X, but got Y" in Go. This should be mentioned in my eBook on Go (which I think you have?)

Reply:
I actually copeid this from a test case above :laughing: line: 105.
I will change everything in this file as soon as I get home.
(Sadly not yet, but it is on my "must buy" list, I am about to move to a bigger apartment so now every penny counts. :( )

Reply:
I'd be pleased to see all the occurrences changed then

Reply:
`tc.Expected` -> `tc.Want`